packages/chargebee/README.md

```diff
@@ -0,0 +1,108 @@
+# `@goodhood/chargebee`
+
+> React chargebee components
+
+## Install
+
+```
+npm i @goodhood/chargebee
+```
+
+Install peer dependencies
+```
+npm i react // v16.x.x
+npm i react-load-script // v0.x.x
+npm i prop-types // v15.x.x
+npm i @babel/runtime // v7.x.x
+```
+
+## Include:
+
+```js
+import { Action, PortalLink, Checkout } from '@goodhood/chargebee';
+import PortalLik from '@goodhood/chargebee/lib/portal_link';
+import Action from '@goodhood/chargebee/lib/action';
+import Checkout from '@goodhood/chargebee/lib/checkout';
+import { SECTION_INVOICES, SECTION_EDIT_SUBSCRIPTION, SECTION_BILLING_ADDRESS } from '@goodhood/chargebee/lib/constants';
```

Comment:
I don't think we should provide examples on how to get files from `lib`. Since we have tree shaking running everywhere where it matters, we don't really need it anymore. I think we should only import from `@goodhood/chargebee`